### DISCUSSION TITLE
Find a freelancer to make a tour app for a travel agency using organic maps

### DISCUSSION THREAD

**Discussion by ivaanmehrotra** [organicmaps/organicmaps#1974] [Q&A]
Hi,

I am trying to find a freelancer to make a tour app for a travel agency using organic maps. 

Idea is to modify the organic maps so that we can show a list of tours when the app loads and when a tour is selected the map loads with the route already built. The user then clicks the start button to get a turn by turn navigation to different places in the tour. When the user reaches a place in the tour, they should be shown a window with details of that place with a continue tour button to continue navigating to the next places in the tour.

I am really not sure if this is the right place to ask this question or if what I am trying to do is even legal and acceptable to the organic maps team. Please guide me accordingly.

**Comment by biodranik** [comment]
Apache 2.0 license allows you to create forks, but you can not use our name "Organic Maps" and our logo.

**Reply by ivaanmehrotra** [reply]
Thanks, I will keep that in mind and also add credits in the credits section. 

@biodranik do you have any recommendations for anyone who can help me with this project?

If there is anyone who reads this and will like to work on this project, please email me at manu.mehrotra.yt@gmail.com to discuss more.

**Reply by ivaanmehrotra** [reply]
Also, does organic maps have support similar to https://developers.google.com/maps/documentation/urls/get-started#directions-action where on accessing a URL the app opens with the route prepopulated and ready to navigate to?

**Reply by biodranik** [reply]
> Also, does organic maps have support similar to https://developers.google.com/maps/documentation/urls/get-started#directions-action where on accessing a URL the app opens with the route prepopulated and ready to navigate to?

Yes, but it should be tested properly.

**Reply by ivaanmehrotra** [reply]
@biodranik can you please point me to some documentation or examples for the URL support mentioned above.

**Reply by biodranik** [reply]
https://github.com/organicmaps/api-ios
https://github.com/organicmaps/api-android

We didn't test it yet. Something may not work properly.

**Reply by ivaanmehrotra** [reply]
@biodranik Thanks! perfect! that was very helpful... I tried - mapswithme://route?sll=50.183933,8.942871&saddr=Start%20Point&dll=49.998912,8.278198&daddr=EndPoint&type=vehicle and it worked well and opened up organicmaps with the route prepopulated and ready to start. However, if I replace mapswithme:// with om:// then it opens the app but the route is not shown so it might be a small bug.

**Reply by biodranik** [reply]
Thanks, we'll take a look. Did you test Android or iOS?

CC @rtsisyk @vng

**Reply by ivaanmehrotra** [reply]
I tested with iOS. Unfortunately, I don't have access to android at hand to test it there. I will report the result if I get my hands on one tomorrow.

**Reply by ivaanmehrotra** [reply]
@biodranik Also, I was able to specify saddr and daddr for source and destination. But, is there a way to specify the way points in between source and destination using the URL too? This will be immensely helpful for my usecase. Thanks for all the help...

**Reply by biodranik** [reply]
Probably not yet,it should be implemented.

**Reply by matkoniecz** [reply]
@biodranik 

> Apache 2.0 license allows you to create forks, but you can not use our name "Organic Maps" and our logo.

is there some place where it is stated (mostly about logo)? I was trying to find some official logo info - including canonical root version and licensing info and this comment and image in https://organicmaps.app/ header is closest I got so far.

(asking as I am reviewing image files in StreetComplete where Organic Maps is presented as part of OSM ecosystem worth investigating, so it falls under fair use I guess)

**Reply by biodranik** [reply]
We won't have any objections to using our logo with the reference to our app/project. That comment was about app forks.
